app/assets/stylesheets/application.scss.erb

```diff
@@ -1016,3 +1016,57 @@ td.day .calendar-text {
   opacity: 0.7;
   background: $state-success-bg;
 }
+
+<% if TeSS::Config.feature['sticky_navbar'] %>
+.navbar {
+  border-radius: 0;
+  font-size: larger;
+  padding-top: 20px;
```

Comment:
See other comment.

If you add an additional class to the header when the sticky navbar feature is enabled, you can just add the CSS rules to that class and they will be ignored if the feature is not enabled.